### ISSUE TITLE
Vendor autoprefixer not working

### ISSUE THREAD

**Issue by tommypyatt** [SnowdogApps/magento2-frontools#327]
Vendor prefixes do not seem to be applying when `styles` task is run with `--prod` flag.

System: Mac OS
Node versions tested: v6.11.2 and v8.11.2

To replicate:

Run `gulp styles --prod` with the following themes config...

```
{
  "blank": {
    "src": "vendor/snowdog/theme-blank-sass",
    "dest": "pub/static/frontend/Snowdog/blank",
    "locale": ["en_GB"],
    "ignore": [".test"]
  }
```

Followed by `grep -c 'ms-flex-order' ../../../pub/static/frontend/Snowdog/blank/en_GB/css/styles.min.css`

This command should return `1` (the string 'ms-flex-order' should be present at least once, as the blank-sass theme contains multiple `order` properties).

However, it returns `0`.

This can also be seen in the browser, when viewing the media gallery on the product detail page:

![screen shot 2018-08-09 at 15 19 49](https://user-images.githubusercontent.com/2505431/43904893-bc44231e-9be7-11e8-94f0-4e1333b8c72e.png)

which should be

![screen shot 2018-08-09 at 14 59 05](https://user-images.githubusercontent.com/2505431/43903751-01f537e8-9be5-11e8-9aa4-6da633d00444.png)

This problem exists in 1.5.11 and is not present in 1.5.10.

**Comment by Igloczek** [comment]
The only thing changed between the versions you mentioned, that somehow can affect CSS processing is dependencies upgrade, that includes bump of Autoprefixer from version "~7.2.1" to "~8.0.0".

Could you try to manually downgrade back to v7 and test it out?

**Comment by tommypyatt** [comment]
I actually just checked the same thing and yes, manually downgrading to 7.2.6 does seem to fix the issue.

**Comment by Igloczek** [comment]
Yeah, I checked it too, it looks like in v8 the `browser` param for some reason is `undefined` instead of taking some default value, which probably leads to ignoring prefixing.

**Comment by Igloczek** [comment]
I fixed it in 1.5.15, there is new config file `browserlist.json`, that takes config in a format described here https://github.com/browserslist/browserslist#full-list

By default it stuff like `order` will not be prefixed, due to browserlist settings, b/c `not dead` exclude browsers like IE, that have more than 0.5% of the market, but are not maintained anymore. So to have IE support you need to create `browserlist.json` in the same directory as `themes.json` and puth there browser range that fits your needs.

**Comment by tommypyatt** [comment]
I was trying to find the answer but it seems you are way faster than me. Thanks for your help @Igloczek.
